File: AF/1.py

number=[1,2,3,4]
t=7
class Solution:
#def twoSum(self, nums: List[int], target: int) -> List[int]:
	def twoSum(self,nums,target):
		length = len (nums)
		for i in range(length):
			mid=target-nums[i]
			for j in range(length):
				if nums[j]==mid and i!=j:
					return [i,j]
		return False
alex=Solution()
print(alex.twoSum(number,t))

File: AF/1.c
int* twoSum(int* nums, int numsSize, int target) {
    int i=0;
    int*new;
    new=(int*)malloc(2*sizeof(int));
    for(i<0;i<numsSize;i++){
        int j=0;
        for(j<0;j<i;j++){
            if(nums[i]+nums[j]==target){
                
                new[0]=i;
                new[1]=j;
                return new;
            }
        }
    }
    return 0;
}